src/main/java/org/apache/maven/plugins/dependency/tree/TreeMojo.java

```diff
@@ -267,6 +306,10 @@ public void execute()
         {
             throw new MojoExecutionException( "Cannot serialise project dependency graph", exception );
         }
+        catch ( DependencyResolutionException e )
+        {
+            e.printStackTrace();
```

Comment:
stack trace might not be very helpful since this is an external issue, not a code issue; is there a logger we can report to instead?